Makefile

```diff
@@ -74,16 +74,12 @@ api/%/zz_generated.deepcopy.go: bin/controller-gen $(DEEPCOPY_GEN_INPUTS)
 
 MANAGER_BIN_INPUTS=$(shell find ./controllers ./api ./pkg -name "*mock*" -prune -o -name "*test*" -prune -o -type f -print) main.go go.mod go.sum
 .PHONY: build
-build: binaries generate-mocks generate-deepcopy manifests release-manifests ## Build manager binary.
+build: binaries generate-deepcopy lint manifests release-manifests ## Build manager binary.
```

Comment:
@jweite-amazon Fixup makefile for generation of deepcopy files.

Actually pretty happy that I removed the generated files and thus created this bug. Means that the deepcopy stuff wasn't properly set in the Makefile, but that we were just typically getting lucky that the deepcopy stuff was present and not broken.